tests/Resources/ti.ui.attributedstring.test.js

```diff
@@ -4,10 +4,14 @@
  * Licensed under the terms of the Apache Public License
  * Please see the LICENSE included with this distribution for details.
  */
+/* global OS_ANDROID, OS_IOS, OS_VERSION_MAJOR */
```

Comment:
<!--
  0 failure: 
  1 warning:  tests/Resources/t...
  
  
  DangerID: danger-id-Danger;
  File: tests/Resources/ti.ui.attributedstring.test.js;
  Line: 7;
-->

- :warning: tests/Resources/ti.ui.attributedstring.test.js line 7 – 'OS_ANDROID' is defined but never used. (no-unused-vars)